src/globalStyles/global.scss

```diff
@@ -122,9 +122,10 @@ th.reactable-header-sort-asc:after {
 }
 
 .table tbody tr td {
-  padding:6px 10px !important;
+  padding:3px 10px !important;
   border-top:none;
 }
 
-
-
+.table {
+  margin-bottom: 5px;
+}
```

Comment:
Let's maybe do patient view/table specific. Just to keep the bootstrap styles intact